sci-libs/libecpint/libecpint-1.0.6-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Know it's not your package, but could you EAPI 8, given sci@ is "quiet"?

Reply:
Done.